tests/integration/targets/digital_ocean_database/tasks/main.yml

```diff
@@ -0,0 +1,90 @@
+---
+- block:
+
+    - name: Ensure API key is provided
+      ansible.builtin.fail:
+        msg: do_api_key should be defined in tests/integration/integration_config.yml
+      when:
+        - do_api_key is not defined
+        - do_api_key | length == 0
+
+    - name: Ensure database is absent
+      community.digitalocean.digital_ocean_database:
+        oauth_token: "{{ do_api_key }}"
+        state: absent
+        name: "{{ database_name }}"
+        region: "{{ do_region }}"
+        engine: "{{ database_engine }}"
+        size: "{{ database_size }}"
+      register: result
+
+    - name: Verify database is absent
```

Comment:
This could fail if the DB was left from a previous run. 
You may want to do two deletes and only check the result of the last one